nvflare/app_common/workflows/cross_site_eval.py

```diff
@@ -34,17 +34,27 @@ def __init__(
         validation_timeout: int = 6000,
         server_models=[DefaultCheckpointFileName.GLOBAL_MODEL],
         participating_clients=None,
-        num_clients: int = 2,
```

Comment:
@SYangster please help check if this is needed, or it is ok to remove as in this PR

Reply:
This is ok to remove in this PR since we have `participating_clients`